pkgs/by-name/si/simple64/package.nix

```
@@ -91,6 +94,26 @@ stdenv.mkDerivation (finalAttrs: {
     runHook postInstall
   '';
 
+  desktopItems = [
+    (makeDesktopItem {
+      name = "simple64";
+      desktopName = "simple64";
+      genericName = "Nintendo 64 Emulator";
+      type = "Application";
+      exec = "simple64-gui";
+      mimeTypes = [ "application/x-n64-rom" ];
+      icon = fetchurl {
+        url = "https://raw.githubusercontent.com/simple64/simple64/v${finalAttrs.version}/simple64-gui/icons/simple64.svg";
+        hash = "sha256-d5TEq4spbAOQoUKH2COxDUO3DweGBiCQKAS/6izJ54s=";
+      };
```

```suggestion
      icon = "simple64";
```
it is not recommended to have store paths in `.desktop` files.

The `.desktop` file automatically search the icon at `$out/share/icons/hicolor/scalable/apps/$icon.svg`, so you should install the `.svg` file to that location in `intallPhase`.